### PR TITLE
add pages und a fixed 501 table

### PR DESCRIPTION
**Pull request by ChristophLeuker**
I just completed my US1. For this i created a table where the user can write his scores into later. For now there are just dummy datas so the user can see how it will look like.

### CODE REVIEW

File: components/game501/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+
```
Comment (julialoeschel) [inline components/game501/index.js:5]: no blank spaces here :)

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "next": "13.3.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "socket.io": "^4.7.0",
```
Comment (julialoeschel) [inline package.json:19]: do you need this already? 
you dont need to deinstall but for next time: only install things when you use them.

File: pages/_app.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
+import HomePage from ".";
 import GlobalStyle from "../styles";
 
 export default function App({ Component, pageProps }) {
   return (
     <>
       <GlobalStyle />
+
```
Comment (julialoeschel) [inline pages/_app.js:8]: delete empty lines here

File: components/game501/index.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+          </tr>
+          <tr>
+            <TableCell></TableCell>
+            <TableCell></TableCell>
+            <TableCell></TableCell>
+            <TableCell></TableCell>
+          </tr>
+        </tbody>
+      </Table>
+    </Container>
```
Comment (julialoeschel) [inline components/game501/index.js:112]: wow, this looks fancy. 
I am excited to see it deployed!

**Review by julialoeschel — COMMENTED** [review]
Great!! 
I left some comments.
please deploy your app on vercel for QA 

:octocat:

File: components/game501/index.js (hunk 1 above)
Comment (ChristophLeuker) [reply components/game501/index.js:5]: deleted spaces

File: package.json (hunk 2 above)
Comment (ChristophLeuker) [reply package.json:19]: alright. this is just for a possible later use ;)

File: pages/_app.js (hunk 3 above)
Comment (ChristophLeuker) [reply pages/_app.js]: did it

File: components/game501/index.js (hunk 4 above)
Comment (ChristophLeuker) [reply components/game501/index.js:111]: oh yes

**Comment by julialoeschel** [comment]
🚀 lgtm

**Pull request merged**